assets/docs/callmemehdi/models/word2vec_swahili/1.md

```diff
@@ -0,0 +1,97 @@
+# Module callmemehdi/word2vec_swahili/1
+
+This is a word2vec model created using Swahili Wikipedia data
+
+<!-- asset-path: https://gsoc-tf.web.app/swahili_word2vec.tar.gz -->
+<!-- task: text-embedding -->
+<!-- fine-tunable: true -->
+<!-- format: saved_model_2 -->
+<!-- license: apache 2 -->
```

Comment:
Please use `apache-2.0` here since this needs to match a value from tags/license.yaml.